packages/ui/package.json

```diff
@@ -38,14 +38,16 @@
     "prune": "ts-prune src"
   },
   "dependencies": {
-    "@dnd-kit/core": "^5.0.1",
-    "@dnd-kit/sortable": "^6.0.0",
+    "@dnd-kit/core": "^6.0.8",
+    "@dnd-kit/sortable": "^7.0.2",
     "@heroicons/react": "^2.0.13",
     "@scena/react-guides": "^0.16.0",
+    "antd": "^5.9.4",
+    "form-render": "^2.2.16",
     "hotkeys-js": "^3.8.7",
     "pdfjs-dist": "2.12.313",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
```

Comment:
Hey @hand-dot, 

Was there a reason for downgrading the React version?

Reply:
```
npm ls react
root
└─┬ @pdfme/ui@2.2.1 -> ./packages/ui
  ├─┬ @dnd-kit/core@6.0.8
  │ ├─┬ @dnd-kit/accessibility@3.0.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @dnd-kit/utilities@3.2.1
  │ │ └── react@16.14.0 deduped
  │ └── react@16.14.0 deduped
  ├─┬ @dnd-kit/sortable@7.0.2
  │ └── react@16.14.0 deduped
  ├─┬ @heroicons/react@2.0.18
  │ └── react@16.14.0 deduped
  ├─┬ @testing-library/react@12.1.5
  │ └── react@16.14.0 deduped
  ├─┬ antd@5.9.4
  │ ├─┬ @ant-design/cssinjs@1.17.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @ant-design/icons@5.2.6
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @ant-design/react-slick@1.0.2
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @rc-component/color-picker@1.4.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @rc-component/mutate-observer@1.1.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @rc-component/tour@1.10.0
  │ │ ├─┬ @rc-component/portal@1.1.2
  │ │ │ └── react@16.14.0 deduped
  │ │ └── react@16.14.0 deduped
  │ ├─┬ @rc-component/trigger@1.17.0
  │ │ ├─┬ rc-align@4.0.15
  │ │ │ └── react@16.14.0 deduped
  │ │ └── react@16.14.0 deduped
  │ ├─┬ qrcode.react@3.1.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-cascader@3.17.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-checkbox@3.1.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-collapse@3.7.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-dialog@9.2.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-drawer@6.4.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-dropdown@4.1.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-field-form@1.38.2
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-image@7.2.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-input-number@8.1.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-input@1.2.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-mentions@2.8.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-menu@9.12.0
  │ │ ├─┬ rc-overflow@1.3.2
  │ │ │ └── react@16.14.0 deduped
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-motion@2.9.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-notification@5.1.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-pagination@3.6.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-picker@3.14.4
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-progress@3.5.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-rate@2.12.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-resize-observer@1.3.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-segmented@2.2.2
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-select@14.9.0
  │ │ ├─┬ rc-virtual-list@3.11.2
  │ │ │ └── react@16.14.0 deduped
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-slider@10.2.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-steps@6.0.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-switch@4.1.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-table@7.34.4
  │ │ ├─┬ @rc-component/context@1.4.0
  │ │ │ └── react@16.14.0 deduped
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-tabs@12.12.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-textarea@1.4.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-tooltip@6.0.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-tree-select@5.13.0
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-tree@5.7.12
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-upload@4.3.5
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-util@5.37.0
  │ │ └── react@16.14.0 deduped
  │ └── react@16.14.0 deduped
  ├─┬ form-render@2.2.16
  │ ├─┬ @ant-design/icons@4.8.1
  │ │ └── react@16.14.0 deduped
  │ ├─┬ ahooks@3.7.5
  │ │ └── react@16.14.0 deduped
  │ ├─┬ rc-color-picker@1.2.6
  │ │ └── react@16.14.0 deduped
  │ ├── react@16.14.0 deduped
  │ ├─┬ virtualizedtableforantd4@1.3.1
  │ │ └── react@16.14.0 deduped
  │ └─┬ zustand@4.4.2
  │   ├── react@16.14.0 deduped
  │   └─┬ use-sync-external-store@1.2.0
  │     └── react@16.14.0 deduped
  ├─┬ react-dom@16.14.0
  │ └── react@16.14.0 deduped
  └── react@16.14.0
```
I made sure to use the same React version as the other libraries.